internal/core/legacy/endpoints/audit.go

```diff
@@ -307,7 +307,7 @@ func getPermissionBody(listReq *apistructs.AuditsListRequest, identityInfo apist
 		pcr.Scope = apistructs.SysScope
 	} else {
 		pcr.Scope = apistructs.OrgScope
-		pcr.ScopeID = uint64(listReq.OrgID)
+		pcr.ScopeID = listReq.OrgID[0]
```

Comment:
I think you should through an error if multiple orgID provided in org scope.

Reply:
has been fixed